src/transformers/models/cohere/modeling_cohere.py

```diff
@@ -1114,6 +1114,7 @@ def forward(
         return_dict: Optional[bool] = None,
         cache_position: Optional[torch.LongTensor] = None,
         num_logits_to_keep: int = 0,
+        **loss_kwargs,
```

Comment:
I think making it an argument `loss_args` rather than kwargs would be better?

Reply:
They are explicitly `kwargs`, not positional so kwargs is more accurate

Reply:
I wasn’t referring to the variable name. My suggestion is to use loss_kwargs instead of **loss_kwargs. The current design prevents forward functions from accepting additional keyword arguments, which could be inconvenient.

Reply:
We haven't accepted that at all before

Reply:
This is prone to evolve either way, with #33932 as an example!